timescale-forge/backup-restore-cloud.md

```diff
@@ -0,0 +1,20 @@
+# Backup and restore
+Timescale Cloud has a range of automated backup and restore mechanisms. All
+automated backups in Timescale Cloud are created using the `pgbackrest` tool.
+There is no need for you to manually perform backups for your Timescale Cloud
+service.
+
+Timescale Cloud automatically creates one full backup every week. We also create
+incremental backups every day, which store any changes since the last full
+backup. The two most recent full backups are stored securely on our Timescale
```

Comment:
> along with the most recent incremental backups

and all changes in-between. Before going to the database, each database modification is first appended to a so-called write-ahead log. That write-ahead log is cut in segments 16MB each, called WAL segments. WAL segments and can be used to "replay" the server restored from the initial backup to an arbitrary point in time. We store WAL all necessary WAL segments together with the backups.